server/src/test/java/com/datastrato/gravitino/server/web/rest/TestPartitionOperations.java

```diff
@@ -211,6 +211,46 @@ public void testListPartitionNames() {
     Assertions.assertTrue(errorResp2.getMessage().contains("test exception"));
   }
 
+  @Test
+  public void testListPartitions() {
```

Comment:
Can you please add a test case returning empty partitions?

Reply:
added to IT